src/main/java/com/blackducksoftware/integration/hub/alert/channel/hipchat/HipChatChannel.java

```diff
@@ -73,6 +81,41 @@ public String getApiUrl() {
         return HIP_CHAT_API;
     }
 
+    @Override
+    public String testGlobalConfig(final GlobalHipChatConfigEntity entity) {
+        if (entity == null) {
```

Comment:
Maybe we should add a test method to the above RestDistributionChannel class? Theoretically, all rest channel classes will also have a their associated test method

Reply:
Slack doesn't. Each test will be specific to the rest api that is being tested. We can abstract it when it actually makes sense to abstract it.